README.md

```diff
@@ -7,11 +9,55 @@ as you would any other component of the application.
 If you are unsure of what should go into the `readme` let this article,
 written by an experienced Chingu, be your starting point -
 [Keys to a well written README](https://tinyurl.com/yk3wubft).
+```
 
-And before we go there's "one more thing"! Once you decide what to include
-in your `readme` feel free to replace the text we've provided here.
+# MediPhase
 
-> Own it & Make it your Own!
+
+It's inevitable that at some point in your life you or a loved one will have to undergo some type of surgery - either as an inpatient or an outpatient. In this event you or a companion, depending on who's having the surgery, will be in a hospital waiting room while the surgery is being performed.
+
+Enter MediPhase. MediPhase helps medical teams monitor a patient's surgery journey with ease. MediPhase provides real-time updates on surgical patient status. 
+
+Checked In, Pre-Procedure, In Progress, Closing, Recovery and Complete. 
+
+Simple, secure, and compassionate care at your medical team's fingertips.
+
+## Overview 
+
+As part of Chingu Voyage 56, our team is building MediPhase. 
+
+## Features 
+
+### UI/UX 
+// TODO add UML Diagrams and Figma Images here 
+
+
+### Screens
+- [ ] Home Screen 
+- [ ] Patient Information Screen
+- [ ] Patient Status Update Screen
+- [ ] Patient Status Display Screen 
+- [ ] Validation and Error Handling
+
+
+## Folder Structure 
+
+The app uses the following folder structure: 
+
+`/src` - All React source code files in there
+`/src/screens` - In this folder, have a subfolder for each visual screen
+`/src/components` - Any reusable components that will be used in multiple components/screens
+`/src/hooks` - React hooks that we make
+
+## Storage
+* As a more frontend-focused app, we're using local storage for any data that needs to be persisted. 
+* If any cross-component data sharing is required, we can use React Context API for now. 
```

Comment:
I'd tweak this and say to show that we are actually using it (which we are, as of my PR for signing in lol)
> we're using React Context API for now